src/Model/Command/UserRecommendation.php

```diff
@@ -239,6 +265,13 @@ protected function getCommandType(): string
         return 'user-based-recommendations';
     }
 
+    protected function serializeBoosts(): array
```

Comment:
```suggestion
    protected function getSerializedBoosts(): array
```

To make it clear from the name the method does not have sideffects.

Reply:
fixed